server/services/store/sqlstore/workspaces.go

```diff
@@ -145,3 +146,62 @@ func (s *SQLStore) GetWorkspaceCount() (int64, error) {
 	}
 	return count, nil
 }
+
+func (s *SQLStore) GetUserWorkspaces(userID string) ([]model.UserWorkspace, error) {
+	var query sq.SelectBuilder
+
+	if s.dbType == mysqlDBType {
+		query = s.getQueryBuilder().
+			Select("Channels.ID", "Channels.DisplayName", "COUNT(focalboard_blocks.id)").
+			From("focalboard_blocks").
+			Join("ChannelMembers ON focalboard_blocks.workspace_id = ChannelMembers.ChannelId").
+			Join("Channels ON ChannelMembers.ChannelId = Channels.Id").
+			Where(sq.Eq{"ChannelMembers.UserId": userID}).
+			Where(sq.Eq{"focalboard_blocks.type": "board"}).
+			Where(sq.Like{"focalboard_blocks.fields": "%\"isTemplate\":false%"}).
+			GroupBy("Channels.Id", "Channels.DisplayName")
+	} else {
+		query = s.getQueryBuilder().
+			Select("channels.id", "channels.displayname", "count(focalboard_blocks.id)").
+			From("focalboard_blocks").
+			Join("channelmembers ON focalboard_blocks.workspace_id = channelmembers.channelid").
+			Join("channels ON channelmembers.channelid = channels.id").
+			Where(sq.Eq{"channelmembers.userid": userID}).
+			Where(sq.Eq{"focalboard_blocks.type": "board"}).
+			Where("focalboard_blocks.fields ->> 'isTemplate' = 'false'").
```

Comment:
This is not going to work on SQLite. I guess you have to use the previous block for SQLite.

Reply:
This is only intended to run as a plugin, which doesn't use SQLite.